keyboards/lazydesigners/the40/keymaps/via/keymap.c

```diff
@@ -16,29 +16,32 @@
 
 #include QMK_KEYBOARD_H
 
+#define LT2_TAB LT(2, KC_TAB)
```

Comment:
Just a heads up, VIA doesn't support LT very well.

Reply:
Sorry, deleted the LT part. Thanks!